webui/src/Components/ConfirmExportModal.tsx

```diff
@@ -14,9 +16,12 @@ interface ConfirmExportModalProps {
 
 export const ConfirmExportModal = forwardRef<ConfirmExportModalRef, ConfirmExportModalProps>(
 	function ConfirmExportModal(props, ref) {
+		const { userConfig } = useContext(RootAppStoreContext)
+
 		const [data, setData] = useState<string | null>(null)
 		const [show, setShow] = useState(false)
 		const [format, setFormat] = useState(ExportFormatDefault)
+		const [filename, setFilename] = useState(String(userConfig.properties?.default_export_filename))
```

Comment:
I think this should be set in the `show` function (line 63), so that it gets reset to the default each time this export modal gets shown

Reply:
I think my latest commit should fix this, let me know if you had a different implementation idea